### PR TITLE
Remove unfinished write transactions

### PR DESCRIPTION
**Pull request by Kubuxu**
This needs to be updated in fs-repo-migrations

### CODE REVIEW

File: convert.go [hunk 1]
```diff
@@ -110,8 +112,38 @@ func Move(oldPath string, newPath string, out io.Writer) error {
...
+
+			names, err := indir.Readdirnames(-1)
+			if err != nil {
+				return err
+			}
+
+			for _, n := range names {
+				p := filepath.Join(oldPath, n)
+				// part of unfinished write transaction
+				// remove it
```
Comment (kevina) [inline convert.go]: I am not 100% convinced this is a good thing to do.  The file could contain important data.  I could be wrong but this seams like the type of thing that would end up in the "lost+found" directory if this where a traditional Unix filesystem.  Maybe we could move the file into into a special directory.  @whyrusleeping what do you think.

**Comment by kevina** [comment]
@Kubuxu could you do me a favor and fix this line:

> log.Warningf("found unexpected file in datastore directory: \"%s\", moving anyway", fn)

so that there is a newline at the end?

File: convert.go (hunk 1 above)
Comment (whyrusleeping) [inline convert.go]: My original suggestion was to delete it. I still think that is the right thing to do. The write that created this file failed, giving the user no expectation of the data existing anywhere on disk.

Comment (kevina) [inline convert.go]: Okay, I'm fine with that.

**Pull request merged**